contracts/ibc-reflect-send/src/msg.rs

```diff
@@ -0,0 +1,133 @@
+#![allow(clippy::field_reassign_with_default)] // see https://github.com/CosmWasm/cosmwasm/issues/685
+
+use cosmwasm_std::{Coin, ContractResult, CosmosMsg, Empty, HumanAddr};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::state::AccountData;
+
+/// InitMsg just needs to know the code_id of a reflect contract to spawn sub-accounts
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InitMsg {}
```

Comment:
```suggestion
#[serde(rename_all = "snake_case")]
pub struct InitMsg {}
```

For later when we come up with fields or people copy/paste from here?